app/livechat/server/lib/QueueManager.js

```diff
@@ -27,8 +27,8 @@ export const QueueManager = {
 		const { rid } = message;
 		const name = (roomInfo && roomInfo.fname) || guest.name || guest.username;
 
-		const room = LivechatRooms.findOne(createLivechatRoom(rid, name, guest, roomInfo));
-		let inquiry = LivechatInquiry.findOne(createLivechatInquiry(rid, name, guest, message));
+		const room = LivechatRooms.findOneById(createLivechatRoom(rid, name, guest, roomInfo));
```

Comment:
I feel a little bit uncomfortable adding these changes to the PR as I have no idea what it is or how to test it 😬

Reply:
> I feel a little bit uncomfortable adding these changes to the PR as I have no idea what they are or how to test them 😬

Yeah, You're right..
We were facing an issue when running the `Enterprise` version because of some changes we did there, so to be able to build a new docker image I implemented the fixes here.
But I'm moving this code to another branch, sry.

Reply:
Alright, thanks! :D